Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apk --no-cache update && \
         tzdata \
         yarn \
         zip
+RUN apk update && apk add g++ gcc libxml2 libxml2-dev libxslt-dev
```

Comment:
could you move these bulkrax dependencies into the previous RUN statement? it'll help reduce the number of layers in the container